src/read.c

```diff
@@ -3215,7 +3624,8 @@ static avifResult avifParse(avifDecoder * decoder)
         avifROData boxContents = AVIF_DATA_EMPTY;
 
         // TODO: reorg this code to only do these memcmps once each
-        if (!memcmp(header.type, "ftyp", 4) || !memcmp(header.type, "meta", 4) || !memcmp(header.type, "moov", 4)) {
+        if (!memcmp(header.type, "ftyp", 4) || !memcmp(header.type, "meta", 4) || !memcmp(header.type, "moov", 4) ||
+            !memcmp(header.type, "coni", 4)) {
```

Comment:
Can we put `!memcmp(header.type, "coni", 4)` inside `#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)`?